extensions/HuggingFace/python/requirements.txt

```diff
@@ -17,4 +17,5 @@ torchaudio
 transformers # Used for text generation
 
 #Other
-asyncio
\ No newline at end of file
+asyncio
+scipy
```

Comment:
let's add this to the `#Hugging Face Libraries - Local Inference Tranformers & Diffusors` section of this file and say `# Used for audio`